concordium-sdk/src/main/java/com/concordium/sdk/crypto/wallet/web3Id/Request.java

```diff
@@ -0,0 +1,17 @@
+package com.concordium.sdk.crypto.wallet.web3Id;
+
+import java.util.List;
+
+import com.concordium.sdk.crypto.wallet.web3Id.Statement.RequestStatement;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.extern.jackson.Jacksonized;
+
+@Getter
+@Builder
+@Jacksonized
+public class Request {
```

Comment:
Request is very generic. It would nice if it was a bit more specific.

Reply:
I'll leave it as is, unless we find a good name. :thinking: